### PR TITLE
YugabyteDB benchbase support

### PR DESCRIPTION
**Pull request by shaharuk-yb**
- Added Yugabyte run spec, config files, yb collector classes and DDLs
- minor bug fixes

### CODE REVIEW

File: config/postgres/sample_seats_config.xml [hunk 1]
```diff
@@ -44,4 +44,4 @@
             <name>UpdateReservation</name>
         </transactiontype>
     </transactiontypes>
-</parameters>
+</parameters>
```
Comment (hbhanawat) [inline config/postgres/sample_seats_config.xml:47]: No change here. Please revert the file.

File: pom.xml [hunk 2]
```diff
@@ -112,6 +112,26 @@
...
+        <profile>
+            <id>yugabyte</id>
+            <properties>
+                <classifier>yugabyte</classifier>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>com.yugabyte</groupId>
+                    <artifactId>jdbc-yugabytedb</artifactId>
+                    <version>42.3.0</version>
```
Comment (hbhanawat) [inline pom.xml:124]: Please use the latest version [42.3.4](https://mvnrepository.com/artifact/com.yugabyte/jdbc-yugabytedb/42.3.4)

File: pom.xml [hunk 3]
```diff
@@ -112,6 +112,26 @@
...
+                </dependency>
+
+                <!-- https://mvnrepository.com/artifact/com.zaxxer/HikariCP -->
+                <dependency>
+                    <groupId>com.zaxxer</groupId>
+                    <artifactId>HikariCP</artifactId>
+                    <version>4.0.3</version>
```
Comment (hbhanawat) [inline pom.xml:131]: Why is HikariCP needed?

File: src/main/java/com/oltpbenchmark/api/BenchmarkModule.java [hunk 4]
```diff
@@ -221,7 +222,7 @@ public final void createDatabase() throws SQLException, IOException {
     public final void createDatabase(DatabaseType dbType, Connection conn) throws SQLException, IOException {
 
             ScriptRunner runner = new ScriptRunner(conn, true, true);
-
+
```
Comment (hbhanawat) [inline src/main/java/com/oltpbenchmark/api/BenchmarkModule.java:225]: No changes here. Please revert this file.

**Review by GargiChakraverty-yb — APPROVED** [review]

File: src/main/java/com/oltpbenchmark/api/Procedure.java [hunk 5]
```diff
@@ -120,8 +120,6 @@ else if (is != null) {
         else {
             pStmt = conn.prepareStatement(stmt.getSQL());
         }
-
```
Comment (hbhanawat) [inline src/main/java/com/oltpbenchmark/api/Procedure.java:123]: Please try avoiding these changes. These changes create unnecessary traces in git blame

File: src/main/java/com/oltpbenchmark/api/collectors/DBParameterCollectorGen.java [hunk 6]
```diff
@@ -30,6 +30,8 @@ public static DBParameterCollector getCollector(DatabaseType dbType, String dbUr
                 return new PostgresCollector(dbUrl, username, password);
             case COCKROACHDB:
                 return new CockroachCollector(dbUrl, username, password);
+            case YUGABYTE:
+                return new YuagbyteCollector(dbUrl, username, password);
```
Comment (hbhanawat) [inline src/main/java/com/oltpbenchmark/api/collectors/DBParameterCollectorGen.java:34]: As discussed with Rakshit yesterday, we do not need  YugabyteCollector. We can use PostgresCollector for now. Let's not add code which is not needed.

File: src/main/java/com/oltpbenchmark/api/collectors/YuagbyteCollector.java [hunk 7]
```diff
@@ -0,0 +1,25 @@
+package com.oltpbenchmark.api.collectors;
```
Comment (hbhanawat) [inline src/main/java/com/oltpbenchmark/api/collectors/YuagbyteCollector.java:1]: remove this file as mentioned in previous comment

**Review by hbhanawat — APPROVED** [review]

**Pull request merged**